### ISSUE TITLE
Including Templates - name mistmatch

### ISSUE THREAD

**Issue by weaverryan** [schmittjoh/JMSTwigJsBundle#1]
Hey Johannes!

Not really an issue probably, more of question. How should I be including other templates?

In order for the assetic twig_js filter to compile, I need to use the Symfony2 template format:

```
{% include 'AcmeDemoBundle:Default:foo.html.twig' %}
```

But to actually include it in assetic, I'm using the file-path format:

```
"@AcmeDemoBundle/Resources/views/Default/foo.html.twig"
```

On the frontend, my template is assigned to the variable `foo.html`. However, the rendered `include` looks like this:

```
`new AcmeDemoBundle.Default.foo.html(...`
```

In other words, the name of the template is mis-matched. So, am I doing something wrong In either including the template or how I include the template?

Thanks!

**Comment by schmittjoh** [comment]
It seems like a bug.

Does it work if you explictly define the template name using `{% twig_js name="foo" %}`?

**Comment by weaverryan** [comment]
Wow, yes, that worked perfectly. So, in this example, the template that's being included "foo.html.twig" has this at the top:

```
{% twig_js name="foo" %}
```

When rendering the template server-side, that tag is ignored. But when compiling to JS, "foo" becomes the function name for this template, and when including it from other templates correctly via `{% include 'AcmeDemoBundle:Default:foo.html.twig' %}`, twig js properly converts that into a call to the "foo" template. Just repeating with more detail for anyone else's future reference.

Thanks Johannes!
